src/azure-cli/azure/cli/command_modules/sql/_params.py

```diff
@@ -224,6 +225,18 @@ def get_location_type_with_default_from_resource_group(cli_ctx):
     options_list=['--object-id', '-i'],
     help='The unique ID of the Azure AD administrator.')
 
+read_scale_param_type = CLIArgumentType(
```

Comment:
Would you mind mark your new parameters as Preview using https://github.com/Azure/azure-cli/blob/dev/doc/authoring_command_modules/authoring_commands.md#preview-commands-and-arguments?

Reply:
This functionality is in full release, not preview.  Can you let me know why we should mark this as preview?

Reply:
In case you want to change your commands. But if you are sure that you will not change your commands, I am fine to it.

Reply:
We cannot change these commands as they are in full release.